lineman/app/models/base_records_interface.coffee

```diff
@@ -45,8 +46,20 @@ angular.module('loomioApp').factory 'BaseRecordsInterface', (RestfulClient, $q)
     fetchByKey: (key) ->
       @restfulClient.getMember(key)
 
-    fetch: (params) ->
-      @restfulClient.getCollection(params)
+    fetch: ({params, path, cacheKey}) ->
```

Comment:
Collapsed this into a single method which I actually found really nice to write fetch methods with (in coffee at least):

```
@fetch
  path: '/my_custom_path'
  params:
    param1: value1
    param2: value2
  cacheKey 'myCustomCacheKey'
```